agent_payload.proto

```diff
@@ -72,7 +72,10 @@ message SketchPayload {
                         int64 ts = 1;
                         int64 n = 2;
                         double min = 3;  
-                        repeated Entry entries = 4 [(gogoproto.nullable) = false];
+                        double max = 4;
+                        double avg = 5;
+                        double sum = 6;
+                        repeated Entry entries = 7 [(gogoproto.nullable) = false];
```

Comment:
just a note (probably not a real issue right now since this is not in production): using a different protobuf tag for `Entry` (i.e. changing it from `4` to `7`) is a backward-incompatible change for the protobuf payload.